client/src/App.js

```diff
@@ -1,9 +1,10 @@
-import React from "react";
+import React from 'react';
+import SendEmail from './pages';
 
 function App() {
   return (
     <div className="App">
-      <h1>Taskillo</h1>
+      <SendEmail />
```

Comment:
Good Job, Nada
I think here you should group all elements front send email in one page ```sendEmailPage``` and call it here.

> Call it from the pages folder